dlt/destinations/impl/databricks/sql_client.py

```diff
@@ -37,7 +38,9 @@ def __init__(self, dataset_name: str, credentials: DatabricksCredentials) -> Non
 
     def open_connection(self) -> DatabricksSqlConnection:
         conn_params = self.credentials.to_connector_params()
-        self._conn = databricks_lib.connect(**conn_params, schema=self.dataset_name)
+        self._conn = databricks_lib.connect(
```

Comment:
this is my only issue here: could we support 2.9 driver? the 3.x has
```
paramstyle = "named"
```
and 2.9 pyformat so it is trivial to distinguish them
why: I bet there are people with constrained environments (ie Airflow) where upgrading dependencies is hard. and 3.0 was released only in march

Reply:
2.9 still works if you have it installed. There's no change in our code except this argument that's ignored in 2.9.   
`paramstyle` doesn't matter with `use_inline_params`, this makes 3.x work like 2.x which does not use parametrized queries at all, it's some custom escaping and `%s` string formatting under the hood. :man_shrugging:

Reply:
If we want to switch to named paramstyle we can detect driver version and cluster version. But I don't know enough about it to tell if this is reliable in all environments: https://docs.databricks.com/en/sql/language-manual/functions/current_version.html